app/controllers/api/v1/salons_controller.rb

```diff
@@ -1,13 +1,16 @@
 module Api
   module V1
     class SalonsController < ApplicationController
-      before_action :set_salon, only: %i[show update destroy] 
+      before_action :set_salon, only: %i[show update destroy]
 
       attr_accessor :salon
 
       def index
-        @salons = Salon.all
-        render json: @salons
+        render json: params[:sort] ? Salon.order(sort_params.to_h) : Salon.all
```

Comment:
please separate logic from ternary operator and extract it to normal variables to improve readability.